src/plugins/workspace/public/plugin.ts

```diff
@@ -547,7 +557,17 @@ export class WorkspacePlugin
       });
     }
 
-    return {};
+    registerDefaultCollaboratorTypes({
+      getStartServices: core.getStartServices,
+      collaboratorTypesService: this.collaboratorTypes,
+    });
+
+    return {
+      setCollaboratorTypes: (types: WorkspaceCollaboratorType[]) => {
+        this.collaboratorTypes.setTypes(types);
+      },
```

Comment:
I think we can expose the entire collaboratorTypes service.
```suggestion
      collaboratorTypes: this.collaboratorTypes;
      },
```